tests/helper.js

```diff
@@ -244,6 +244,7 @@ module.exports = {
 
   waitForAngularComplete: () => {
     return browser.wait(() => {
+      console.warn('browser.AngularComplete() should be doing this. Start replacing and see if this is required');
```

Comment:
Did you mean to leave this in here? It seems like something you have probably verified through other tests now, making this warning no longer necessary.

Reply:
I left this here because I saw that  `waitForAngularComplete` is used throughout our tests quite a bit. Was hoping to warn everyone so they see this will likely not work and the proper replacement.

Reply:
Do you think it should be removed @craig-landry ?